analytics/models.py

```
@@ -1,49 +1,56 @@
+        unique_together = ("ip_address", "user_agent")
+
     def most_common_url(self):
         try:
-            return PageView.objects.filter(ip_address=self.ip_address).order_by('-timestamp').first().url
+            return (
+                PageView.objects.filter(ip_address=self.ip_address)
+                .order_by("-timestamp")
+                .first()
+                .url
+            )
         except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)